install/install.sh

```diff
@@ -50,11 +50,10 @@ docker --version || curl -fsSL https://get.docker.com | env -u VERSION sh || (
 )
 systemctl enable docker
 
-#creating a Unix group called docker and adding users to it.
-echo "Created the docker group"
+# creating a Unix group called docker and adding users to it
```

Comment:
This comment also needs to change.
Something like "Add user to 'docker' group allowing docker commands without root privilegies"